pyms/DPA/PairwiseAlignment.py

```diff
@@ -183,7 +183,7 @@ def align(a1: Alignment, a2: Alignment, D: float, gap: float) -> Alignment:
 	return ma
 
 
-def score_matrix(a1: Alignment, a2: Alignment, D: float) -> Alignment:
+def score_matrix(a1: numpy.ndarray, a2: numpy.ndarray, D: float) -> Alignment:
```

Comment:
Sorry, I think this should return numpy.ndarray. Can you also remove the :rtype: flag from the docstring?